README.md

```diff
@@ -0,0 +1,175 @@
+For Node.js Developers
+======================
+
+The JSON Schema Test Suite is meant to be a language agnostic test suite for testing JSON Schema validation libraries.
+It is generally added to projects as a git submodule. However, to simplify things for Node.js developers, the test suite has also
+been made available as an npm package.
+
+    npm install json-schema-test-suite
+
+### Usage:
+
+There are a number of ways to load tests from the suite:
+
+    const testSuite = require('json-schema-test-suite');
+
+    // this will load all (required and optional) draft4 tests
```

Comment:
Does this actually still load draft4?

Reply:
Yes it does, however I've now taken the opportunity to make it load draft 6 by default. I've updated the README accordingly